frontend/src/components/phones/onboarding/RealPhoneSetup.tsx

```diff
@@ -1,8 +1,6 @@
 import { ChangeEventHandler, FormEventHandler, useState } from "react";
 import { Localized, useLocalization } from "@fluent/react";
 import { isPossiblePhoneNumber } from "react-phone-number-input";
-import PhoneInput from "react-phone-number-input/input";
-import { E164Number } from "libphonenumber-js/min";
```

Comment:
Nice, that's almost a complete dependency removed :broom: As far as I can see, its only remaining use is via `isPossiblePhoneNumber`, which actually is re-exported from libphonenumber-js. So you could remove `react-phone-number-input` completely and replace it by just that dependency. But according to the docs that function only verifies phone number length, so maybe we could even rip that entire dependency out and just check that ourselves?

Reply:
I did - removed import, now we just have to clean up package.json, I'm hesitant to update it because last time I recall there was an issue due to not using Volta?

Reply:
That should be resolved now (Kaitlyn now installed Volta as well, and I believe you already had it?), and if not, we'll want to resolve it rather than never changing our dependencies anymore :P So feel free to remove it, if something goes wrong we can deal with that then.